app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -220,6 +237,24 @@ public void bind(NavDrawerData.DrawerItem drawerItem) {
                         .load();
             }
         }
+
+        private void updateLayoutParams(TextView feedTitle, int textColor) {
+            lastParams = feedTitle.getLayoutParams();
+            RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(
+                    ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT);
+            params.addRule(RelativeLayout.ALIGN_LEFT, R.id.imgvCover);
+            params.addRule(RelativeLayout.ALIGN_RIGHT, R.id.imgvCover);
+            params.addRule(RelativeLayout.BELOW, R.id.imgvCover);
+            feedTitle.setLayoutParams(params);
+            feedTitle.setEllipsize(TextUtils.TruncateAt.END);
+            feedTitle.setSingleLine(true);
+            feedTitle.setTextColor(textColor);
```

Comment:
This is done in both branches of the `if` statement, so it can be moved directly to the `bind` method for clarity